operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationService.java

```diff
@@ -193,6 +193,9 @@ default Optional<InformerStoppedHandler> getInformerStoppedHandler() {
         log.error("Fatal error in informer: {}. Stopping the operator", informer, ex);
         System.exit(1);
       } else {
+        // In case informer is stopped on start (not synced) the controller will stop automatically.
+        // The case when already synced and stopped without an exception, can happen only if
+        // informer is stopped explicitly.
         log.debug("Informer stopped: {}. Error: {}", informer, ex);
```

Comment:
Perhaps some of this information should be put more explicitly in the log message?

Reply:
updated!